.github/workflows/main.yaml

```diff
@@ -53,3 +55,25 @@ jobs:
             /tmp/docker/setup-hw-image
             /tmp/docker/setup-hw-secret-image
           retention-days: 1
+  push_image:
+    name: push_image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Push images to  if tagged
+        if: startsWith( github.ref, 'refs/tags/v' )
```

Comment:
`if` can be specified directly for jobs.
```yaml
    push_image:
      if: github.event_name != 'pull_request'
      needs: build
```

Reply:
Also, you don't need to match the tag name with `v*`.
The tag is already filtered.